app/controllers/items_controller.rb

```diff
@@ -4,9 +4,13 @@ def new
     type = params[:type]
     druid = params[:druid]
     targets = params[:targets]
+    raise 'missing required params' unless type && druid && targets
```

Comment:
I'm pretty sure you can here. The params don't need to be directly related to a model. 

From http://edgeguides.rubyonrails.org/action_controller_overview.html#strong-parameters

> In addition, parameters can be marked as required and will flow through a predefined raise/rescue flow that will result in a 400 Bad Request being returned if not all required parameters are passed in.